3rd_party/CMakeLists.txt

```diff
@@ -31,6 +31,6 @@ get_directory_property(CMAKE_CXX_FLAGS DIRECTORY marian-dev DEFINITION CMAKE_CXX
 set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} PARENT_SCOPE)    
 set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} PARENT_SCOPE)    
 
-if(COMPILE_PYTHON)
-  add_subdirectory(pybind11)
-endif(COMPILE_PYTHON)
+#if(COMPILE_PYTHON)
+#  add_subdirectory(pybind11)
+#endif(COMPILE_PYTHON)
```

Comment:
Looks like you moved this to a `find_package()` call later on. I think you can safely just remove it here then, no need to just comment it out.